src/wp-admin/admin-header.php

```diff
@@ -90,7 +90,11 @@
 <title><?php echo esc_html( $admin_title ); ?></title>
 <?php
 
-wp_enqueue_style( 'colors' );
+// We do not need the default admin CSS stylesheets in the Site Editor.
```

Comment:
```suggestion
// Do not load the default admin CSS stylesheets if in the Site Editor.
```

That's a little more into the point imho.